test/fops/t.c

```diff
@@ -35,6 +35,9 @@ const char *uri;				/* Object */
 const char *config;				/* Object config */
 
 static FILE *logfp;				/* Log file */
+bool explicit_transaction;			/* Operations within explicit
```

Comment:
I'd prefer this be called `use_txn` both because the term `explicit` says to me that if we don't use it we get `implicit` transactions, which is not the case and because it is so long that the comment structure is very awkward.